entries/QUnit.module.xml

```diff
@@ -15,16 +15,31 @@
 				<desc>Runs after each test</desc>
 			</property>
 		</argument>
+		<argument name="grouped_tests" optional="true">
+			<desc>A function with grouped tests</desc>
```

Comment:
This is not a good description. Could replace "function" with "callback", and actually explain what "grouped tests" means.